.tekton/kueue-0.11-pull-request.yaml

```diff
@@ -0,0 +1,618 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/openshift/kubernetes-sigs-kueue?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/cancel-in-progress: "false"
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main"
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: kueue-main
+    appstudio.openshift.io/component: kubernetes-sigs-kueue-main
+    pipelines.appstudio.openshift.io/type: build
+  name: kubernetes-sigs-kueue-0-11-on-push
+  namespace: kueue-operator-tenant
+spec:
+  params:
+  - name: git-url
+    value: '{{source_url}}'
+  - name: revision
+    value: '{{revision}}'
+  - name: output-image
+    value: quay.io/redhat-user-workloads/kueue-operator-tenant/kubernetes-sigs-kueue-0-11:{{revision}}
+  - name: build-platforms
+    value:
+    - linux/x86_64
+  - name: dockerfile
+    value: upstream/release-0.11/Dockerfile
```

Comment:
We can't build upstream docker files. We have to use our RHEL ones.

Reply:
That is our dockerfile